UI/Selenium/NLog.config

```diff
@@ -3,7 +3,13 @@
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
 
```

Comment:
Are you using this file to control nlog settings or appsettings?  You have duplicate settings in both and ideally it should all be in appsettings, or else here if forced to do so.

Reply:
NLog.config now removed. All settings are now managed through appsettings.json